packages/angular/webpack.dev.config.js

```diff
@@ -0,0 +1,505 @@
+  "plugins": [
+    new NoEmitOnErrorsPlugin(),
+    new CopyWebpackPlugin([
+      {
+        "context": "src",
+        "to": "",
+        "from": {
+          "glob": "assets\\**\\*",
+          "dot": true
+        }
+      }
+    ], {
+      "ignore": [
+        ".gitkeep",
+        "**/.DS_Store",
+        "**/Thumbs.db"
+      ],
+      "debug": "warning"
+    }),
+    new ProgressPlugin(),
+    new CircularDependencyPlugin({
+      "exclude": /(\\|\/)node_modules(\\|\/)/,
+      "failOnError": false,
+      "onDetected": false,
+      "cwd": projectRoot
+    }),
+    new NamedLazyChunksWebpackPlugin(),
+    new HtmlWebpackPlugin({
+      "template": "./src\\index.html",
+      "filename": "./index.html",
+      "hash": false,
+      "inject": true,
+      "compile": true,
+      "favicon": false,
+      "minify": false,
+      "cache": true,
+      "showErrors": true,
+      "chunks": "all",
+      "excludeChunks": [],
+      "title": "Webpack App",
+      "xhtml": true,
+      "chunksSortMode": function sort(left, right) {
+        let leftIndex = entryPoints.indexOf(left.names[0]);
+        let rightIndex = entryPoints.indexOf(right.names[0]);
+        if (leftIndex > rightIndex) {
+            return 1;
+        }
+        else if (leftIndex < rightIndex) {
+            return -1;
+        }
+        else {
+            return 0;
+        }
+    }
+    }),
+    new BaseHrefWebpackPlugin({}),
+    new CommonsChunkPlugin({
+      "name": [
+        "inline"
+      ],
+      "minChunks": null
+    }),
+    new CommonsChunkPlugin({
+      "name": [
+        "vendor"
+      ],
+      "minChunks": (module) => {
+                return module.resource
+                    && (module.resource.startsWith(nodeModules)
+                        || module.resource.startsWith(genDirNodeModules)
+                        || module.resource.startsWith(realNodeModules));
+            },
+      "chunks": [
+        "main"
+      ]
+    }),
+    new SourceMapDevToolPlugin({
+      "filename": "[file].map[query]",
+      "moduleFilenameTemplate": "[resource-path]",
+      "fallbackModuleFilenameTemplate": "[resource-path]?[hash]",
+      "sourceRoot": "webpack:///"
+    }),
+    new CommonsChunkPlugin({
+      "name": [
+        "main"
+      ],
+      "minChunks": 2,
+      "async": "common"
+    }),
+    new NamedModulesPlugin({}),
+    new AngularCompilerPlugin({
+      "mainPath": "main.ts",
+      "platform": 0,
+      "hostReplacementPaths": {
+        "environments\\environment.ts": "environments\\environment.ts"
+      },
+      "sourceMap": true,
+      "tsConfigPath": "src\\tsconfig.app.json",
+      "skipCodeGeneration": true,
+      "compilerOptions": {}
+    })
+  ],
+  "node": {
+    "fs": "empty",
+    "global": true,
+    "crypto": "empty",
+    "tls": "empty",
+    "net": "empty",
+    "process": true,
+    "module": false,
+    "clearImmediate": false,
+    "setImmediate": false
+  },
+  "devServer": {
+    "historyApiFallback": true
```

Comment:
could you add 
```
  hot: false,
  inline: false
```
under `devServer` and try this project out on the `nos://` protocol in the nOS client :D

Reply:
I add these properties and it works. The problem is that only the "Get Neo Address" functionality works. All other buttons throw and error in the console. Could you check it out?

Reply:
I'll spin up nOS-Local and give it a try tonight